### ISSUE TITLE
Update schema for Solr 7

### ISSUE THREAD

**Issue by jakubklimek** [linkedpipes/dcat-ap-viewer#49]
The current schema creation command fails with:
```
{
  "responseHeader":{
    "status":0,
    "QTime":151},
  "errors":[{
      "add-field":{
        "name":"modified",
        "type":"tdate",
        "docValues":true,
        "multiValued":false},
      "errorMessages":["Field 'modified': Field type 'tdate' not found.\n"]},
    {
      "add-field":{
        "name":"issued",
        "type":"tdate",
        "docValues":true},
      "errorMessages":["Field 'issued': Field type 'tdate' not found.\n"]},
    {
      "delete-copy-field":{
        "source":"*",
        "dest":"_text_"},
      "errorMessages":["Copy field directive not found: '*' -> '_text_'\n"]},
    {
      "add-field":{
        "name":"temporal-start",
        "type":"tdate",
        "docValues":true},
      "errorMessages":["Field 'temporal-start': Field type 'tdate' not found.\n"]},
    {
      "add-field":{
        "name":"temporal-end",
        "type":"tdate",
        "docValues":true},
      "errorMessages":["Field 'temporal-end': Field type 'tdate' not found.\n"]}]}
```

**Comment by jakubklimek** [comment]
This works:
```
curl https://[url]/solr/dcat-ap-viewer/schema -X POST -H 'Content-type:application/json' --data-binary '{
    "add-field" : { "name" : "iri", "type" : "string" , "indexed" : false },
    "add-field" : { "name" : "modified", "type" : "pdate", "docValues" : true , "multiValued" : false},
    "add-field" : { "name" : "issued", "type" : "pdate", "docValues" : true },
    "add-field" : { "name" : "accrualPeriodicity", "type" : "string" },
    "add-field" : { "name" : "description", "type" : "string" },
    "add-field" : { "name" : "publisher", "type" : "string" , "indexed" : false },
    "add-field" : { "name" : "publisherName", "type" : "string" },
    "add-field" : { "name" : "title", "type" : "string", "docValues" : true },
    "add-field" : { "name" : "format", "type" : "strings", "indexed" : false },
    "add-field" : { "name" : "formatName", "type" : "strings" },
    "add-field" : { "name" : "license", "type" : "strings", "indexed" : false },
    "add-field" : { "name" : "keyword", "type" : "strings" },
    "add-copy-field" : { "source" : "description", "dest" : "_text_" },
    "add-copy-field" : { "source" : "title", "dest" : "_text_" },
    "add-copy-field" : { "source" : "keyword", "dest" : "_text_" },
    "add-field" : { "name" : "temporal-start", "type" : "pdate", "docValues" : true },
    "add-field" : { "name" : "temporal-end", "type" : "pdate", "docValues" : true },
    "add-field" : { "name" : "spatial", "type" : "string" }
}'
```

And this is also required:
```
curl https://[url]/solr/dcat-ap-viewer/config -H 'Content-type:application/json' -d'{
    "set-property" : {"requestDispatcher.requestParsers.enableRemoteStreaming":true},
    "set-property" : {"requestDispatcher.requestParsers.enableStreamBody":true}
}'
```

**Comment by skodapetr** [comment]
We should update the readme.md file with this information + target version of Solr.

**Comment by jakubklimek** [comment]
Closed by d5f6a574e8319210492ee8d3fbb9ff89b2674963
